_inc/client/setup-wizard/feature-toggle/map-feature-to-props.js

```diff
@@ -0,0 +1,1055 @@
+					} );
+				},
+			};
+		},
+	},
+
+	'post-by-email': {
+		mapStateToProps: state => {
+			return {
+				title: __( 'Post by email' ),
+				details: __(
+					'Post by email is a quick way to publish new posts without visiting your site.'
+				),
+				checked: getSetting( state, 'post-by-email' ),
+				configureLink: '#/traffic?term=post%20by%20email',
+			};
+		},
+		mapDispatchToProps: dispatch => {
+			return {
+				onToggleChange: currentCheckedValue => {
+					return dispatch( updateSettings( { 'post-by-email': ! currentCheckedValue } ) );
+				},
+			};
+		},
+	},
+
+	publicize: {
+		mapStateToProps: state => {
+			const siteRawUrl = getSiteRawUrl( state );
+
+			return {
+				title: __( 'Publicize' ),
+				details: __( 'Automaticaly share content on your favorite social media accounts.' ),
+				checked: getSetting( state, 'publicize' ),
+				configureLink: getRedirectUrl( 'calypso-marketing-connections', { site: siteRawUrl } ),
+				isLinkExternal: true,
+			};
+		},
+		mapDispatchToProps: dispatch => {
+			return {
+				onToggleChange: currentCheckedValue => {
+					return dispatch( updateSettings( { publicize: ! currentCheckedValue } ) );
+				},
+			};
+		},
+	},
+
+	'related-posts': {
+		mapStateToProps: state => {
+			return {
+				title: __( 'Related posts' ),
+				details: __(
+					'Keep your visitors engaged with related content at the bottom of each post.'
+				),
+				checked: getSetting( state, 'related-posts' ),
+				optionsLink: '#/settings?term=related%20posts',
+			};
+		},
+		mapDispatchToProps: dispatch => {
+			return {
+				onToggleChange: currentCheckedValue => {
+					return dispatch( updateSettings( { 'related-posts': ! currentCheckedValue } ) );
+				},
+			};
+		},
+	},
+
+	scan: {
+		mapStateToProps: state => {
+			const vaultPressData = getVaultPressData( state );
+			const isScanEnabled =
+				true === get( vaultPressData, [ 'data', 'features', 'security' ], false );
+
+			const sitePlan = getSitePlan( state );
+			const planClass = getPlanClass( sitePlan.product_slug );
+
+			const inCurrentPlan = [ 'is-premium-plan', 'is-business-plan', 'is-scan-plan' ].includes(
+				planClass
+			);
+
+			let optionsLink;
+			if ( inCurrentPlan ) {
+				optionsLink = '#/settings?term=scan';
+			}
+
+			let upgradeLink;
+			if ( ! inCurrentPlan ) {
+				upgradeLink = '#/plans';
+			}
+
+			let info;
+			if ( inCurrentPlan ) {
+				info = getInfoString( sitePlan.product_name );
+			}
+
+			return {
+				title: __( 'Security scanning' ),
+				details: __( 'Stop threats to keep your website safe.' ),
+				checked: isScanEnabled,
+				isDisabled: inCurrentPlan,
+				isPaid: true,
+				optionsLink,
+				upgradeLink,
+				info,
+			};
+		},
+	},
+
+	search: {
+		mapStateToProps: state => {
+			let upgradeLink;
+			let optionsLink;
+			const sitePlan = getSitePlan( state );
+			if (
+				'is-business-plan' !== getPlanClass( sitePlan.product_slug ) &&
+				! hasActiveSearchPurchase( state )
+			) {
+				upgradeLink = '#/plans';
```

Comment:
Shouldn't we redirect to jetpack.com's landing page for this, since it has details about the exact feature, and not just a list of plans?

Reply:
@jeherve We had a long discussion about this and there are some limitations with the routes for redirecting.  With the short timeline we don't have time to refactor it so as a team we decided to send users to the Jetpack plans page. Thanks for bringing this up, normally we'd want to send users to LP's.